e2e-tests/custom-login/conf.js

```diff
@@ -14,7 +14,14 @@
 const daemonUtil = require('../tools/daemon-util');
 const commonConfig = require('../tools/common-config');
 
-module.exports.config = commonConfig.configure(Promise.all([
-  daemonUtil.startCustomLoginServer(),
-  daemonUtil.startResourceServer()
-]));
+// Start the resource server only for implicit flow
+if (process.env.TEST_TYPE === 'implicit') {
+  module.exports.config = commonConfig.configure(Promise.all([
+    daemonUtil.startCustomLoginServer(),
+    daemonUtil.startResourceServer()
+  ]));
+} else {
+  module.exports.config = commonConfig.configure(Promise.all([
```

Comment:
nit: Spacing

Reply:
Where should I add spacing?

Reply:
Line `24` needs two spaces: 
```
else {
  module.exports.config = commonConfig.configure(Promise.all([
    ...
}
```

I believe this is also in the other file's `else` statement